Sources/Controllers/2.0/Settings/OAImportDuplicatesViewController.mm

```diff
@@ -0,0 +1,518 @@
+//
+//  OAImportDuplicatesViewControllers.m
+//  OsmAnd Maps
+//
+//  Created by nnngrach on 15.10.2020.
+//  Copyright © 2020 OsmAnd. All rights reserved.
+//
+
+#import "OAImportDuplicatesViewController.h"
+#import "OAImportCompleteViewController.h"
+#import "Localization.h"
+#import "OAColors.h"
+#import "OAResourcesUIHelper.h"
+#import "OASettingsImporter.h"
+#import "OAApplicationMode.h"
+#import "OAQuickActionRegistry.h"
+#import "OAQuickActionType.h"
+#import "OAQuickAction.h"
+#import "OAMapSource.h"
+#import "OASQLiteTileSource.h"
+#import "OAPOIUIFilter.h"
+#import "OAAvoidRoadInfo.h"
+#import "OAProfileDataObject.h"
+#import "OAMenuSimpleCell.h"
+#import "OAMenuSimpleCellNoIcon.h"
+#import "OATitleTwoIconsRoundCell.h"
+#import "OAActivityViewWithTitleCell.h"
+
+#define kMenuSimpleCell @"OAMenuSimpleCell"
+#define kMenuSimpleCellNoIcon @"OAMenuSimpleCellNoIcon"
+#define kTitleTwoIconsRoundCell @"OATitleTwoIconsRoundCell"
+#define kCellTypeWithActivity @"OAActivityViewWithTitleCell"
+#define RENDERERS_DIR @"rendering/"
+#define ROUTING_PROFILES_DIR @"routing/"
+
+
+@interface HeaderType : NSObject
+@property (nonatomic) NSString *label;
+@end
+
+@implementation HeaderType
+- (instancetype) initWithLabel:(NSString *)label
+{
+    self = [super init];
+    if (self)
+    {
+        _label = label;
+    }
+    return self;
+}
+@end
+
+
+@interface OAImportDuplicatesViewController () <UITableViewDelegate, UITableViewDataSource, OASettingsImportExportDelegate>
+
+@end
+
+@implementation OAImportDuplicatesViewController
+{
+    OsmAndAppInstance _app;
+    NSArray<id> *_duplicatesList;
```

Comment:
```NSArray<id>``` is very wrong! Either specify a type (if possible) or leave out the braces completely.